modules/gcp/dc/new_domain_admin_user.ps1.tpl

```diff
@@ -43,45 +43,45 @@ function Decrypt-Credentials {
     $headers.Add("Authorization", "Bearer $($token)")
 
     try {
+        "--> Decrypting account_password..."
         $resource = New-Object "System.Collections.Generic.Dictionary[[String],[String]]"
         $resource.Add("ciphertext", "${account_password}")
         $response = Invoke-RestMethod -Method "Post" -Headers $headers -Uri $DECRYPT_URI -Body $resource
         $credsB64 = $response."plaintext"
         $DATA."account_password" = [System.Text.Encoding]::ASCII.GetString([System.Convert]::FromBase64String($credsB64))
     }
     catch {
-        "Error decrypting credentials: $_"
+        "--> ERROR: Failed to decrypt credentials: $_"
         return $false
     }
 }
 
 Start-Transcript -path $LOG_FILE -append
 
 if ([string]::IsNullOrWhiteSpace("${kms_cryptokey_id}")) {
-    "Not using encryption"
+    "--> Script is not using encryption for secrets."
 } else {
-    "Using ecnryption key ${kms_cryptokey_id}"
+    "--> Script is using encryption key ${kms_cryptokey_id} for secrets."
     Decrypt-Credentials
 }
 
-Write-Output "================================================================"
-Write-Output "Creating new AD Domain Admin account ${account_name}..."
-Write-Output "================================================================"
-
+"================================================================"

```

Comment:
Didn't you get rid of these lines in the other files? having said that, I think I like to have section headers look a little different rather than just "-->" so it's easier to spot where we're at in the logs

Reply:
I think I ended up keeping these lines and just removed Write-Output for consistency sake.